luaconf.h

```diff
@@ -872,9 +883,16 @@ static inline int time(void *p)
 
 /* stdlib.h */
 #include <linux/slab.h>
-#define free(a) 			kfree(a)
+#define free(a) 		kfree
 #define realloc(a, b) 		krealloc(a, b, GFP_ATOMIC)
```

Comment:
offtop: `GFP_NOWAIT` looks sufficient, isn't it?

Reply:
I believe it wouldn't be enough for nflua and xdplua, as GFP_NOWAIT is likely to fail.. however, this is something that I would like to change soon.. the API client should be able to choose the allocator properly and the default should (if any) should be `GFP_KERNEL`.. I'm working on updating to Lua 5.4 currently.. after that I should do some clean up / refactoring.. I'll include this.. thanks!